Update sharepoint documents configurepermissions-document (#678)
SuperOfficeDocs/superoffice-docs: docs/en/document/cloud/sharepoint-documents/configure-permissions-documents.md

# How to set up SharePoint permissions for documents

> [!NOTE]

> > This feature is only relevant if 'Confidential activities' is enabled for users in 'Settings and maintenance', and 'Visible for' field is set on the the document property:

Reviewer: ```suggestion
> This feature is relevant only if **Confidential activities** is enabled for users in Settings and maintenance, and the **Visible for** field is set on the document property:
```

<details>

> <summary>Show details for Confidential activities and Visible for</summary>

Reviewer: We aim to use this syntax less and I recommend omitting it. If it's worth mentioning, there's no need to hide the info.

Reviewer 2: @digitaldiina  It was Frode's edits. I just asked you for a review. 

Anyway, I changed the <details> section to be visible and part of the Note. Is it ok now? 

Thanks for the info.

Reviewer: Thanks @pavlas77 and @fbhub2 :)

> 'Confidential activities' in the [Settings and maintenance - Users][1]: activates the 'Visible for' for the user.

Reviewer: ```suggestion
* **Confidential activities** in the [Settings and maintenance - Users][1] activates the *Visible for* for the user.
```

> 'Visible for' field in the [document dialog][2]: display a list that allows you to choose whether the document will be visible to all users, private (i.e. visible only to the user defined as owner), or visible to all users in one of the user groups that the owner belongs to. 

Reviewer: ```suggestion
* The **Visible for** field in the [document dialog][2] display a list that allows you to choose whether the document will be visible to all users, private (visible only to the user defined as owner), or visible to all users in one of the user groups that the owner belongs to. 
```

​![Review SharePoint site groups​ -screenshot][img4]

<!-- Referenced links -->

> [1]: https://help.superoffice.com/Documentation/Help/EN/CRM/10.1/WebHelpAdmin/chap07/Adding_users_web.htm

Reviewer: We've migrated the admin guide to docs. Suggest you link directly to the most recent version of the **Add user** page.

```suggestion
[1]: ../../../admin/user-management/learn/add-associate.md
```
